operator/src/main/java/oracle/kubernetes/operator/steps/ManagedServersUpStep.java

```diff
@@ -15,6 +15,7 @@
 import java.util.stream.Collectors;
```

Comment:
Update all changed files to have copyright date changed to 2022.

Reply:
Changed in ec20ac8.